package.json

```diff
@@ -95,7 +95,8 @@
     "test": "TZ=UTC jest --verbose --no-cache",
     "test:single": "jest --verbose -w 1",
     "build": "webpack --config config/prod.webpack.config.js",
-    "verify": "npm-run-all build lint test"
+    "verify": "npm-run-all build lint test",
+    "dev:provisioning": "DEV_ROUTES_PATH=config/routes.dev.config.js ./devel/provisioning.sh"
```

Comment:
Could we make the routes configuration optional? maybe on a flag, or two npm scripts?
I see value in both consuming the local backend and backend from stage, but if I were to choose, I'd go with using the stage backend and only documenting usage of `DEV_ROUTES_PATH`. If documented I think it's reasonable to be running:
`DEV_ROUTES_PATH=config/routes.dev.config.js npm run dev:provisioning`
to connect to local backends.

Reply:
the stagde backend is by default, we can add in a follow-up PR a flag for routing the local API as well